share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -417,6 +417,14 @@ def postJob (rscApi: RemoteCacheApi) (job: Job) (_wakeroot: String) (hidden: Str
     | rscApiPostJob req
 
 # Determines if the user has requested that debug info be emitted from the cache
-target shouldDebugRemoteCache Unit: Boolean = match (getenv "DEBUG_WAKE_REMOTE_CACHE")
-    Some _ -> True
-    None -> False
+target shouldDebugRemoteCache Unit: Boolean =
+    match (Pair (getenv "DEBUG_WAKE_REMOTE_CACHE") (getenv "DEBUG_WAKE_SHARED_CACHE"))
+        Pair None None -> False
+        Pair (Some _) None ->
```

Comment:
I feel like at some point we'll want to write an official `getBooleanEnv` to unify all of our handling of `DEBUG_WAKE_REMOTE_CACHE=0` explicit falses.